languageservice/src/value-providers/definition.ts

```diff
@@ -71,6 +71,14 @@ function mappingValues(
             // No special insertText in this case
             break;
 
+          case DefinitionType.String:
+            insertText = `\n${indentation}${key}: `;
+            break;
+
+          case DefinitionType.Boolean:
+            insertText = `\n${indentation}${key}: `;
+            break;
+
```

Comment:
```suggestion
          case (DefinitionType.String, DefinitionType.Boolean):
            insertText = `\n${indentation}${key}: `;
            break;
```

Minor, but we could combine these two into a single case statement if we want to make it clear that we treat them the same for completion text